packages/ses/src/compartment.js

```diff
@@ -172,18 +175,63 @@ defineProperties(InertCompartment, {
  * @returns {Compartment['constructor']}
  */
 
+// In order to facilitate migration from the deprecated signature
+// of the compartment constructor,
+//   new Compartent(globals?, modules?, options?)
+// to the new signature:
+//   new Compartment(options?)
+// where globals and modules are expressed in the options bag instead of
+// positional arguments, this function detects the temporary sigil __options__
+// on the first argument and coerces compartments arguments into a single
+// compartments object.
+const compartmentOptions = (...args) => {
+  if (args.length === 0) {
+    return {};
+  }
+  if (
+    args.length === 1 &&
+    typeof args[0] === 'object' &&
+    args[0] !== null &&
+    '__options__' in args[0]
+  ) {
+    const { __options__, ...options } = args[0];
+    assert(
+      __options__ === true,
```

Comment:
since you require it to have value `true`, you should be sure to mention that where you explain `__option__`, such as the doc-comment above. From the doc-comment alone, I would guest that you only tested whether the property is present.